concordium-sdk/src/main/java/com/concordium/sdk/transactions/smartcontracts/WasmModuleVersion.java

```diff
@@ -0,0 +1,25 @@
+package com.concordium.sdk.transactions.smartcontracts;
+
+import com.concordium.sdk.types.UInt32;
+import lombok.Getter;
+
+public enum WasmModuleVersion {
```

Comment:
Documentation. 
Maybe also add that v1 contracts were added as part of protocol 4.